code/src/test/java/com/google/sps/data/ContactStatusTest.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.data;
+
+import org.junit.Test;
+import org.junit.Assert;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+ 
+@RunWith(JUnit4.class)
+public final class ContactStatusTest {
+
```

Comment:
Same as for business test, I would split testMethodsForCheckingBusinessStatus into two seperate methods